### PR TITLE
use keytool service

### PR DESCRIPTION
**Pull request by dsokal**
# Checklist

- [x] I've added an entry to [CHANGELOG.md](https://github.com/expo/eas-cli/blob/main/CHANGELOG.md#main) if necessary. You can comment this pull request with `/changelog-entry [breaking-change|new-feature|bug-fix|chore] [message]` and CHANGELOG.md will be updated automatically.
- [ ] I've tagged the changelog entry with `[EAS BUILD API]` if it's a part of a breaking change to EAS Build API (only possible when updating `@expo/eas-build-job` package).

# Why

We built a keystore service so ppl don't need to install Java to generate keystores.

# How

Use keystore service if:
- keytool is not installed
- keytool failed to generate keystore

# Test Plan

Tested manually.

### CODE REVIEW

**Comment by dsokal** [comment]
/changelog-entry new-feature Use Keystore Service when `keytool` is not installed.

File: packages/eas-cli/src/credentials/android/utils/keystore.ts [hunk 1]
```diff
@@ -94,27 +117,49 @@ async function createKeystoreAsync(
...
+  keyPassword: string;
+  keyAlias: string;
+}
+
+async function createKeystoreInCloudAsync(
+  keystoreParams: KeystoreParams,
+  { showKeytoolDetectionMsg = true } = {}
+): Promise<KeystoreWithType> {
+  if (showKeytoolDetectionMsg) {
+    Log.warn(`Detected that you do not have ${chalk.bold('keytool')} installed locally.`);
```
Comment (brentvatne) [inline packages/eas-cli/src/credentials/android/utils/keystore.ts:142]:
```suggestion
    Log.log(`Detected that you do not have ${chalk.bold('keytool')} installed locally.`);
```

i think this is something that people don't necessarily need to be "warned" about, but rather just told about, and so `log` feels more appropriate than `warn`.

**Review by brentvatne — APPROVED** [review]
👏

File: packages/eas-cli/src/credentials/android/utils/keystore.ts (hunk 1 above)
Comment (dsokal) [reply packages/eas-cli/src/credentials/android/utils/keystore.ts:142]: I wasn't sure what log level to choose but if you also think this shouldn't be a warning then I'm ok with changing it.

**Pull request merged**